src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -115,6 +115,8 @@
     <PlatformManifestFileEntry Condition="'$(PgoInstrument)' != ''" Include="clrjit.pgd" IsNative="true" />
     <PlatformManifestFileEntry Include="libclrjit.so" IsNative="true" />
     <PlatformManifestFileEntry Include="libclrjit.dylib" IsNative="true" />
+    <PlatformManifestFileEntry Include="libclrinterpreter.so" IsNative="true" />
+    <PlatformManifestFileEntry Include="libclrinterpreter.dylib" IsNative="true" />
```

Comment:
Are we going to ship `libclrinterpreter.dylib` on Apple devices? I would expect that we will define `FEATURE_STATICALLY_LINKED` for Apple device platforms and the interpreter is going to be statically linked there.